assume/strategies/flexable.py

```diff
@@ -127,27 +128,74 @@ def get_starting_costs(self, time, unit):
         else:
             return unit.cold_start_cost
 
-    def get_possible_revenues(self, marginal_cost, unit):
-        t = self.current_time
-        price_forecast = []
 
-        if t + self.foresight > unit.price_forecast.index[-1]:
-            price_forecast = unit.price_forecast.loc[t:]
+class flexablePosCRM(BaseStrategy):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # check if kwargs contains crm_foresight argument
+        if "crm_foresight" in kwargs:
+            self.foresight = pd.Timedelta(kwargs["crm_foresight"])
         else:
-            price_forecast = unit.price_forecast.loc[t : t + self.foresight]
+            self.foresight = pd.Timedelta("4h")
```

Comment:
This can also be solved with:
`self.foresight = pd.Timedelta(kwargs.get("crm_foresight", "4h"))`
which does exactly this